src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is work");

```

Comment:
The message in the `doWork` method should be more informative and indicate the type of machine that is working. For example, it could be "Truck started its work."